### PR TITLE
Add more bed models for PrusaSlicer and SuperSlicer

### PR DESCRIPTION
**Pull request by hoangnam123**
#### Which mods/configurations/profiles are added by this PR?
Added 3 additional bed STLs for PrusaSlicer and SuperSlicer. These STLS were imported from the Simplify3D bed verions and fixed the origin to make it work with PS and SS. 

#### How was it tested? 
Simply load the bed models with the software and make sure that the bed shows up and matches the coordinate of the bed plane

### CODE REVIEW

File: slicer_configurations/README.md [hunk 1]
```diff
@@ -28,4 +28,5 @@ like so:
 | hijax_pl_ | [Simplify3D](./Simplify3D/hijax_pl/beds) | Bed images for Simplify3D | :x: | :x: | :heavy_check_mark: | :x: |
 | hartk1213 | [PrusaSlicer V0 Bed and Textures](./PrusaSlicer/hartk1213/V0) | Bed STL and textures for Prusa Slicer | :heavy_check_mark: | :x: | :x: | :x: |
 | koonweee | [SuperSlicer v1.8 Bed](./SuperSlicer/koonweee/beds) | Bed STL for SuperSlicer | :x: | :heavy_check_mark: | :x: | :x: |
+| hoangnam123 | [SuperSlicer v2.4 Bed](./SuperSlicer/hoangnam123) | Bed STL for SuperSlicer | :x: | :x: | :heavy_check_mark: | :x: |
```
Comment (KiloQubit) [inline slicer_configurations/README.md:31]: Please move this line up so it's in alphabetical order.  I realize some lines above it are in the wrong order, but nevertheless please move your line up so it's before `koonweee`.

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for your mod!

Please rename your STL files to remove the space from their names, and see the below comment about the README file.

File: slicer_configurations/README.md (hunk 1 above)
Comment (hoangnam123) [reply slicer_configurations/README.md:30]: i have fixed the order for the other entry also

**Review by KiloQubit — CHANGES_REQUESTED** [review]
Thanks for the updates and fixing the README.  It looks like your STL files are broken, can you make sure they don't have any errors in them?

Screen Shot 2021-04-14 at 14 51 24

**Comment by hoangnam123** [comment]
> 
> 
> Thanks for the updates and fixing the README. It looks like your STL files are broken, can you make sure they don't have any errors in them?
> 
> Screen Shot 2021-04-14 at 14 51 24

Not sure why this matters since this is just the model for viewing only and not for printing. I tried to repair it but it resulted in resetting the origin of the models which doesn't work with PS or SS.

**Comment by KiloQubit** [comment]
Even though we're not printing these STLs, we still cannot accept broken STLs into the repository.  One of the requirements for PRs is

- Submitted STL files are not corrupt. (This can be tested by opening the STL in PrusaSlicer
and checking if mesh errors are reported.)

Also CI fails it (I don't know if you have access to see the output): https://github.com/VoronDesign/VoronUsers/pull/288/checks?check_run_id=2345248160

**Comment by hoangnam123** [comment]
> 
> 
> Even though we're not printing these STLs, we still cannot accept broken STLs into the repository. One of the requirements for PRs is
> 
>     * Submitted STL files are not corrupt. (This can be tested by opening the STL in PrusaSlicer
>       and checking if mesh errors are reported.)
> 
> 
> Also CI fails it (I don't know if you have access to see the output): https://github.com/VoronDesign/VoronUsers/pull/288/checks?check_run_id=2345248160

Let me figure out a way to fix this. I'll abandon this PR for now

**Review by KiloQubit — APPROVED** [review]
Thanks for fixing the STLs!

**Pull request merged**